scripts/openmc-track-to-vtk

```diff
@@ -51,44 +51,20 @@ def main():
     elif not args.out.endswith('.pvtp'):
         args.out += '.pvtp'
 
-    # Import HDF library if HDF files are present
-    for fname in args.input:
-        if fname.endswith('.h5'):
-            import h5py
-            break
-
     # Initialize data arrays and offset.
     points = vtk.vtkPoints()
     cells = vtk.vtkCellArray()
     point_offset = 0
     for fname in args.input:
         # Write coordinate values to points array.
-        if fname.endswith('.binary'):
-            track = open(fname, 'rb')
-
-            # Determine number of particles and tracks/particle
-            n_particles = struct.unpack('i', track.read(4))[0]
-            n_coords = struct.unpack('i'*n_particles, track.read(4*n_particles))
-
-            coords = []
-            for i in range(n_particles):
-                # Read coordinates for each particle
-                coords.append([struct.unpack('ddd', track.read(24))
-                               for j in range(n_coords[i])])
-
-                # Add coordinates to points data
-                for triplet in coords[i]:
-                    points.InsertNextPoint(triplet)
-
-        else:
-            track = h5py.File(fname)
-            n_particles = track['n_particles'].value[0]
-            n_coords = track['n_coords']
-            coords = []
-            for i in range(n_particles):
-                coords.append(track['coordinates_' + str(i + 1)].value)
-                for j in range(n_coords[i]):
-                    points.InsertNextPoint(coords[i][j,:])
+        track = h5py.File(fname)
+        n_particles = track['n_particles'].value
+        n_coords = track['n_coords']
+        coords = []
```

Comment:
This comment is unrelated to this PR, but I wonder if this block of code might be optimized by pre-allocating ``coords`` as a NumPy array outside of the loop rather than appending to a list on each iteration. Not a big deal at all, just couldn't help but express my affinity for NumPy.